alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/YstiaOrchestratorFactory.java

```diff
@@ -47,11 +47,14 @@
 
     @Override
     public YorcPaaSProvider newInstance() {
-        return beanFactory.getBean(YorcPaaSProvider.class);
+        YorcPaaSProvider instance = beanFactory.getBean(YorcPaaSProvider.class);
+        instance.startLogsAndEvents();
+        return instance;
     }
 
     @Override
     public void destroy(YorcPaaSProvider instance) {
+        instance.stopLogsAndEvents();
         // nothing specific, the plugin will be garbaged collected when all references are lost.
```

Comment:
Now there is something specific :smile: we could remove this comment.

Reply:
:smile: